### PR TITLE
[MIG][15.0] account_payment_return

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

File: account_payment_return/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to Invoicing > Sales > Customer Payment Returns, and create a new
```
Comment (DavidJForgeFlow) [inline account_payment_return/README.rst]: Invoicing > Customers > Customer Payment Returns

**Review by DavidJForgeFlow — CHANGES_REQUESTED** [review]
Functional Review :+1:, one change in Readme and done!

**Comment by ValentinVinagre** [comment]
@ChrisOForgeFlow Can you check the comments? thanks 😄

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow Can you check the comments? thanks smile

Done!, thanks

File: account_payment_return/tests/test_payment_return.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import json
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import Form, SavepointCase
+
+
+class TestPaymentReturn(SavepointCase):
```
Comment (victoralmau) [inline account_payment_return/tests/test_payment_return.py]:
```suggestion
from odoo.tests.common import Form, TransactionCase


class TestPaymentReturn(TransactionCase):
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by victoralmau** [comment]
Can you merge your changes in a single commit (migration commit)?

File: account_payment_return/models/payment_return.py [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+            "company_id": self.company_id.id,
+        }
+
+    def _prepare_move_line(self, move, total_amount):
+        self.ensure_one()
+        return {
+            "name": move.ref,
+            "debit": 0.0,
+            "credit": total_amount,
+            "account_id": self.journal_id.default_account_id.id,
```
Comment (carlosdauden) [inline account_payment_return/models/payment_return.py:180]: Use Outstanding Receipts Account

**Review by carlosdauden — CHANGES_REQUESTED** [review]

**Comment by ao-landoo** [comment]
@ChrisOForgeFlow are you continuing with this PR?

**Comment by carlosdauden** [comment]
As there was no response I am preparing a new PR

**Comment by ChrisOForgeFlow** [comment]
> As there was no response I am preparing a new PR
Sorry, I can continue with PR, are you agree?

**Comment by carlosdauden** [comment]
Replaced by this: https://github.com/OCA/account-payment/pull/610

**Pull request closed without merging**